crates/rover-client/src/query/subgraph/check.graphql

```diff
@@ -0,0 +1,48 @@
+  mutation CheckPartialSchemaQuery (
+    $graph_id: ID!
+    $variant: String!
+    $implementingServiceName: String!
+    $partialSchema: PartialSchemaInput!
+  ) {
+    service(id: $graph_id) {
+      checkPartialSchema(
+        graphVariant: $variant
+        implementingServiceName: $implementingServiceName
+        partialSchema: $partialSchema
+      ) {
+        compositionValidationResult {
+          compositionValidationDetails {
+            schemaHash
+          }
+          graphCompositionID
+          errors {
+            message
+          }
+        }
```

Comment:
i don't actually use this anywhere in the actual command. should we be doing composition checks in addition to the `checkSchemaResult`? It's not clear to me how these are intended to interact.

Reply:
Hmm, I think `subgraph check` should be checking for _both_ breaking operations (like `graph check`) _and_ potential composition errors. existing formatting for printing those errors in the apollo cli is [here](https://github.com/apollographql/apollo-tooling/blob/master/packages/apollo/src/commands/service/check.ts#L150-L174). I also do a rudimentary version of printing these errors in the `subgraph delete` command right now. Maybe we should share printing formatting for these two commands (as well as with subgraph push probably)

Reply:
i think i assumed that that would have been part of the check operation in the API! i'll make sure we get this in